Configuration/TypoScript/SeoSitemap/setup.typoscript

```diff
@@ -17,10 +17,10 @@ plugin.tx_seo {
 
 						sortField = datetime
 						lastModifiedField = tstamp
-						pid = {$plugin.tx_news.settings.sitemap.startingpoint}
-						recursive = {$plugin.tx_news.settings.sitemap.recursive}
+						pid = {$plugin.tx_news.settings.sitemap.startingpoint ?? $news.seo.sitemap.startingpoint}
+						recursive = {$plugin.tx_news.settings.sitemap.recursive ?? news.seo.sitemap.recursive}
```

Comment:
Hi there :) I'm just wondering what syntax is actual valid?
`?? $news...` or `?? news...`
With or without $ on the right right of the ??-operator. Or are both valid?

Reply:
The variant with `$` on the right side of `??` operator seems to be the right way: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/Configuration/TypoScript/Syntax/Operators/Index.html#null-coalescing-operator-for-typoscript-constants

Reply:
Thanks for clarifying